frontend/src/pages/auth/styles.module.scss

```diff
@@ -0,0 +1,82 @@
+.auth {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 100vh;
+  padding: 0 5%;
+  overflow: hidden;
+  color: var(--white);
+  background-image: url("images/auth-720x512.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.auth .logo {
+  display: none;
+}
+
+.auth .overlay {
+  position: relative;
+  z-index: 3;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 35%;
+  min-width: 350px;
+  height: 75%;
+  padding: 45px;
+  text-align: left;
+  background-color: var(--white-alpha-20);
+  border-radius: 32px;
+}
+
+.auth-name {
+  position: relative;
+  z-index: 2;
+  margin-bottom: 20vh;
+  text-align: center;
+}
+
+.auth-name .secondary {
+  margin: 0.1px;
+  font-size: 30px;
+}
+
+.auth-name .primary {
+  margin: 0.1px;
+  font-weight: var(--semibold);
+  font-size: 83px;
+}
+
+@media (width < 768px) {
```

Comment:
Let's create an variable something like media-width-tablet

Reply:
@v0ldemar01 default css variables doesn`t work and also when i am trying to do it css litner say 
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/35ddc731-6f12-4962-88e6-bd2951fece08)

Reply:
ok, looks like we can't use vars with media queries, then let's leave it as it is

Reply:
@Ekit18 can we try with this [plugin](https://www.npmjs.com/package/postcss-media-variables)?